railties/lib/rails/generators/app_base.rb

```diff
@@ -69,8 +69,8 @@ def self.add_shared_options_for(name)
         class_option :skip_turbolinks,    type: :boolean, default: false,
                                           desc: 'Skip turbolinks gem'
 
-        class_option :skip_test_unit,     type: :boolean, aliases: '-T', default: false,
-                                          desc: 'Skip Test::Unit files'
+        class_option :skip_test,          type: :boolean, aliases: '-T', default: false,
+                                          desc: 'Skip ActiveSupport::TestCase files'
```

Comment:
I guess there's no need to be super explicit, maybe we could simply say `Skip test files`, wdyt?